ports/stm32f4/peripherals/stm32f4/periph.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2019 Lucian Copeland for Adafruit Industries
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+#ifndef __MICROPY_INCLUDED_STM32F4_PERIPHERALS_PERIPH_H__
+#define __MICROPY_INCLUDED_STM32F4_PERIPHERALS_PERIPH_H__
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#include "stm32f4xx_hal.h"
+#include "stm32f4/pins.h"
+
+// #define PA  0
+// #define PB  1
+// #define PC  2
+// #define PD  3
+// #define PE  4
+// #define PF  5
+// #define PG  6
+// #define PH  7
+// #define PI  8
+// #define PJ  9
+// #define PK  10
+
+// I2C
+
+// Numerical Version
```

Comment:
No need for this if you are using and testing the address version. So please remove.

Reply:
Sure, just wanted to show the alternative as an option, if you felt it'd be better done this way.

Reply:
No worries. Let's make sure and resolve these questions in the draft phase. Just drop a comment on the PR with a mention for me and we can decide.